classes/incident.php

```diff
@@ -194,7 +194,7 @@ public function render($admin=0){
           <small>
               <?php echo _("Impacted service(s): ");
               foreach ( $this->service_name as $key => $value ) {
-                echo '<span class="label label-default">'.$value . '</span>&nbsp;';
+                echo '<span class="label label-default" style="display:inline-block;">'.$value . '</span>&nbsp;';
```

Comment:
Ideally we would want to put this inside the .css file - it would go something like this:
```
.panel .panel-footer .label{
    display: inline-block;
}
```

Reply:
I didn't want to do it in css as I thought it might have been in use somewhere else :) will fix it